src/python/WMComponent/PhEDExInjector/PhEDExInjectorPoller.py

```diff
@@ -351,7 +351,7 @@ def closeBlocks(self):
 
         migratedBlocks = self.getMigrated.execute()
 
-        for siteName in migratedBlocks.keys():
```

Comment:
@amaltaro since you are not changing the way how you are initializing && calling the upper level object [1], why do you want to change the way you iterate here. The only reason I could imagine is later down in this same PR [2]. You obviously want to have a new view on the dictionary, but even with that, keeping the explicit reference to migratedBlocks.keys() here should not break the rest of the code. Have you tested this?  

[1] https://github.com/dmwm/WMCore/blob/master/src/python/WMComponent/PhEDExInjector/PhEDExInjectorPoller.py#L140

[2]
https://github.com/dmwm/WMCore/pull/9438/commits/70567a7dc6dd462f970008751f18a1a886dd2ad3#diff-73e9dd1a91792c7023eeec392d2e28b3R375

Reply:
Iterating through the keys of a dictionary can be either accomplished by `for k in dict.keys():` or `for k in dict:`, but the first call returns a list while the second yields the key on the fly. It uses the same syntax in python 3 too, but the result is different.

Reply:
Thanks @amaltaro!